### DISCUSSION TITLE
Jar file is not opening

### DISCUSSION THREAD

**Discussion by Neferiouscorte** [jagrosh/MusicBot#1789] [Troubleshooting]
I tried everything but its not opening idk why but yesterday it was working perfectly. Anybody had this issue before?

**Comment by vandevsysX64** [comment]
well you need to run in cmd.exe and cd c//example (put where the file is located) then put java -jar jmusicbot-0.4.3.jar if you cant figure it out then go to no text to speeches jmusicbot thing he made he will help you
